CheckboxItem/CheckboxItem.js

```diff
@@ -34,10 +40,10 @@ import componentCss from './CheckboxItem.module.less';
  * Usage:
  * ```
  * <CheckboxItem
- *   defaultSelected={selected}
- *   onToggle={handleToggle}
+ *  defaultSelected={selected}
```

Comment:
Oh, I'm sorry, I did not comment clearly. We use two spaces for indentation, and we need one space just after '*' once. So, we need three spaces here.